docs/Serverless.md

```diff
@@ -0,0 +1,99 @@
+<h1 align="center">Serverless</h1>
+
+Run serverless applications and REST APIs using your existing Fastify application.
+
+> Fastify is not designed to run on the AWS Lambda platform.
+The Fastify framework is designed to make implementing a traditional HTTP/S server easy.
+The AWS Lambda platform services requests in a completely different manner than
+a traditional HTTP/S server and is therefore at odds with Fastify's whole design.
+Regardless, it is possible to use Fastify in this environment if you so choose.
```

Comment:
Change these last few sentences to:
`Regardless, based on the examples given in this document, it is possible to use Fastify in a serverless environment such as AWS Lambda. Again, keep in mind that this is not Fastify's intended use case and we do not test for such integration scenarios.`

Reply:
✅